src/realm/sync/changeset.hpp

```diff
@@ -22,10 +22,7 @@ struct Changeset {
     using file_ident_type = uint_fast64_t;
     using version_type = uint_fast64_t; // FIXME: Get from `History`.
 
-    Changeset();
-    struct share_buffers_tag {
-    };
-    Changeset(const Changeset&, share_buffers_tag);
+    Changeset() = default;
```

Comment:
Now that we don't have any special constructors (i.e. all constructors are default), do we need to have all these explicitly defaulted/deleted constructors at all?

Reply:
There's probably no reason to still have copying deleted. I left it deleted because even though it works it's still probably unintended, but I guess that's just needlessly weird.